### PR TITLE
Updated subscriber landing page

### PR DESCRIPTION
**Pull request by davidrapson**
Subscriber landing page:

- Updated with approved copy
- New design from Ben W
- URL changed to `/subscriber-offer`. TBC, but original URL of `offers/subscriber` conflicted with offers & comps and is also awkward because there is no resource at `offers` so shouldn't be nested.
- Combined `Offer` controller with `Info` as a) this in an information page b) avoids confusion with offers & comps.

**Remaining steps**
- Awaiting high-res version of second image (montage) from Ben W.
- Required images to be uploaded to the Grid instead

**Large screen**
subscriber-large

**Medium (~740px)**
subscriber-740

**Small screen**
subscriber-small

### CODE REVIEW

File: frontend/app/model/Faq.scala [hunk 1]
```diff
@@ -146,4 +146,47 @@ object Faq {
...
+    Item("Can I cancel membership during my trial?",
+      Html("Yes, There is no obligation to continue to a paid Membership after your free 6 months and you can cancel at any time during the trial period."),
+      "cancel-trial"
+    ),
+    Item("I'm a subscriber and I'm also a member, will I get this offer too?",
+      Html(s"Yes, you may be entitled to receive this offer, please call customer services team on 0330 333 6898 or email <a href='mailto:${Config.membershipSupport}'>${Config.membershipSupport}</a>"),
+      "existing-subscriber-member"
+    ),
+    Item("How much does membership cost normally?",
+      Html("The Friend tier of Guardian Membership is free. Partners pay £15 per month or £135 per year if you pay in one go. For Patrons, the cost is £60 per month or £540 per year. If you choose the annual payment you get 3 months free when compared to the monthly price."),
```
Comment (mattandrews) [inline frontend/app/model/Faq.scala]: Don't these pricing values live in a config file somewhere? If so, might be nice to use them here to save cleaning them up if we change prices.

File: frontend/app/model/Faq.scala [hunk 2]
```diff
@@ -146,4 +146,47 @@ object Faq {
...
+      "normal-cost"
+    ),
+    Item("I don’t have an email address, can I still join?",
+      Html("No, you need an email address in order to join Membership. Online registration allows you to access benefits, book tickets and receive regular Membership updates."),
+      "no-email"
+    ),
+    Item("What if my subscription ends during my free Membership trial? ",
+      Html("Your Partner Membership will be unaffected by  any changes made to your current subscription."),
```
Comment (mattandrews) [inline frontend/app/model/Faq.scala]: Double space between "by  and" (probably not visible, but my OCD...)

File: frontend/app/model/Faq.scala [hunk 3]
```diff
@@ -146,4 +146,47 @@ object Faq {
...
+    ),
+    Item("I don’t live in London - can I still be a member?",
+      Html("Yes, Guardian Membership is open to everyone. Guardian Live events at launch will be taking place in London, Edinburgh, Manchester and Bristol, and we have plans to run future events across the UK."),
+      "outside-london"
+    ),
+    Item("I don’t live in the UK - can I still be a member?",
+      Html("Yes, Guardian Membership is open to anyone living anywhere in the world. And although our home is London, we will be extending membership and staging events for members in the USA and Australia."),
+      "outside-uk"
+    ),
+    Item("We will never use your information for any purpose without your permission. You can read more about our approach to data security in our privacy policy h​ere.​",
```
Comment (mattandrews) [inline frontend/app/model/Faq.scala]: It feels like this heading should be a bit shorter...?

File: frontend/app/views/fragments/pricing/offerTable.scala.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+@(tier: com.gu.membership.salesforce.Tier)
+
+@import model.Benefits
+@import views.support.Prices._
+
+@for(pricing <- Benefits.details(tier).pricing) {
+    <table class="table table--offer">
+        <thead>
+            <tr>
+                <td>Pay anually</td>
```
Comment (mattandrews) [inline frontend/app/views/fragments/pricing/offerTable.scala.html]: Typo in `anually` (should be "annually")

**Comment by mattandrews** [comment]
LGTM besides minor comments :+1:

File: frontend/app/model/Faq.scala (hunk 1 above)
Comment (davidrapson) [inline frontend/app/model/Faq.scala]: @mattandrews Yeah, we should do, but currently prices are hardcoded across all FAQ items (not just subscribers) so I think I'm going to look at getting all the prices from the model in another PR.

**Pull request merged**